src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
-
 function App() {
```

Comment:
You can also use arrow function. It's preferable, have shorter syntax. e.g. 
```suggestion
const App = () => <div><p>Example of Functional Component.</p></div>
```

Reply:
Changes are done.